tests/integration/components/common/busy-model-wrapper-test.js

```diff
@@ -22,15 +23,26 @@ moduleForComponent('common/busy-model-wrapper', 'Integration | Component | commo
   }
 });
 
-test('it renders', function(assert) {
-  assert.expect(1);
+test('it shows "saving" if model is saving, but not deleted', function(assert) {
+  let model = { isSaving: true, isDeleted: false };
+  set(this, 'model', model);
+  renderPage();
+
+  assert.equal(page.text, 'Saving...', 'the model is saving');
+
+});
 
+test('it shows "Deleting" if model is saving and is deleted', function(assert) {
+  let model = { isSaving: true, isDeleted: true };
+  set(this, 'model', model);
   renderPage();
 
-  assert.ok(page.isVisible);
+  assert.equal(page.text, 'Deleting...', 'the model is deleting');
 });
 
-test('it shows "saving" if model is saving', function(assert) {
```

Comment:
Have the first two tests passing, by setting the model, however, I'm unsure how to get the this one passing without something to yield. Nikola mentioned earlier that I could set my renderPage to page.render(hbs{{#busy-model-wrapper model=model}}Foo{{/busy-model-wrapper}}) but I've tried this both in my initial renderPage() function and in the test itself and haven't gotten anywhere with it. The error I get is

Error: Element not found.

PageObject: 'page.text'
  Selector: ':first' 

I got this on both of my tests prior, but only because I'd messed up the boolean values onSaving and onDeleting. I fixed that, but I'm not sure what this would need to accept that neither onSaving or onDeleting are false.